assets/airports/kjfk.json

```diff
@@ -7,227 +8,623 @@
+            "draw": [
+                ["COATE*"],
+                ["NEION*"],
+                ["HAAYS*"],
+                ["GAYEL*"],
+                ["DEEZZ*"],
+                ["GREKI*"],
+                ["MERIT*"],
+                ["BAYYS*"],
+                ["BETTE*"],
+                ["HAPIE*"],
+                ["SHIPP*"],
+                ["WAVEY*"],
+                ["WHITE*"]
+            ]
         },
-        "DEEZZ4": {
-            "icao": "DEEZZ4",
-            "name": "DEEZZ FOUR",
+        "MERIT6": {
+            "icao": "MERIT6",
+            "name": "Merit Six",
             "rwy": {
-                "31L": ["_315I238SKORR", "SKORR", ["CESID", "A25+"], "YNKEE"],
-                "31R": ["_315I243SKORR","SKORR", ["CESID", "A25+"], "YNKEE"],
-                "13R": [["_RWY13R3DME", "A5.2+"]],
-                "13L": [["_RWY13L3DME", "A5.2+"]],
-                "4R" : ["_RWY4L3"],
-                "4L" : ["_RWY4L3"],
-                "22R": ["YNKEE"],
-                "22L": ["YNKEE"]
+                "KJFK4L" : ["#099"],
+                "KJFK4R" : ["#099"],
+                "KJFK13L": ["#134"],
+                "KJFK13R": ["#134"],
+                "KJFK22L": ["#224"],
+                "KJFK22R": ["#224"],
+                "KJFK31L": ["CRI", ["_CRI176002", "A25+"], "#176"],
+                "KJFK31R": ["CRI", ["_CRI176002", "A25+"], "#176"]
             },
+            "body": ["MERIT"],
             "exitPoints": {
-                "CANDR": ["DEEZZ", ["HEERO", "A50+"], "KURNL", "CANDR"],
-                "TOWIN": ["DEEZZ", ["HEERO", "A50+"], "KURNL", "TOWIN"]
+                "MERIT": [],
+                "PUT": ["HFD", "PUT"]
             },
-            "draw": [["SKORR", "CESID", "YNKEE"], ["DEEZZ", "HEERO", "KURNL", "CANDR*"], ["DEEZZ", "HEERO", "KURNL", "TOWIN*"]]
+            "draw": [
+                ["MERIT*", "HFD", "PUT*"]
+            ]
         },
-        "SKORR3": {
-            "icao": "SKORR3",
-            "name": "SKORR THREE",
+        "SKORR4": {
+            "icao": "SKORR4",
+            "name": "Score Four",
             "rwy": {
-                "31R": ["_315I243SKORR"],
-                "31L": ["_315I238SKORR"]
+                "KJFK31L": ["_RWY13RSKORR058"],
+                "KJFK31R": ["_RWY13LSKORR063"]
             },
+            "body": ["SKORR"],
             "exitPoints": {
-                "RNGRR": ["SKORR", ["METSS", "A25+"], "RNGRR"],
-                "YNKEE": ["SKORR", ["CESID", "A25+"], "YNKEE"]
+                "SKORR": [],
+                "YNKEE": [["CESID", "A25+"], "YNKEE", "#172"],
+                "RNGRR": [["METSS", "A25+"], "RNGRR", "#223"]
```

Comment:
Should the `#223` be here?

Reply:
Both the FAA and Jeppesen charts are somewhat unclear... to me, I thought the FAA one was inconclusive on whether they meant to show that it was _along_ that radial, or if they meant to fly the radial through RNGRR as the arrow shows. The Jeppesen chart I thought was a little more clearly showing that you should continue along the 223 course, so I went with that. Neither chart had a textual description involving it.

I went back to XPlane (Zibo 737) and actually plugged in `SKORR4 RNGRR RBV J230 COPES J75 GVE` just as an example, and it did not show any `(VECTOR)` legs, just from RNGRR direct to RBV. So my new guess is that they're just illustrating the radial, but don't want you to fly it past RNGRR.

Removed the heading after RNGRR.

![image](https://user-images.githubusercontent.com/5103735/73578454-acc53f80-444d-11ea-955f-ca8be1a196aa.png)

![image](https://user-images.githubusercontent.com/5103735/73578474-bb135b80-444d-11ea-9d23-222f56b1e118.png)

Reply:
Works for me! 👍